### ISSUE TITLE
Performance dashboard

### ISSUE THREAD

**Issue by Typiqally** [Typiqally/epsilon#43]
I propose that the next step for Epsilon is generating the competence matrix shown below. ![image](https://user-images.githubusercontent.com/12190745/201751665-8f8fb429-5353-4d12-ad51-132b2a4a5878.png)

In my opinion, we should make this available for at least Excel and if possible Word, keeping in mind the goal of this project: automating the entire competence document. Optional is LaTeX, which is a niche feature but would still be helpful for some Deltas at least.

So in summary then, create an export pipeline for generating competence matrices together with KPI tables in the following formats:

- [ ] Excel, Must
- [ ] Word, Must if possible
- [ ] LaTeX, Should
- [ ] Others?

**Comment by Typiqally** [comment]
Let me know what you think @NealGeilen

**Comment by NealGeilen** [comment]
So I agree with your reasoning and steps. I would like to test out #42 first. 
Because the documentation states that it has support graphs and other statistics tools 
And the graph part is something the current libraries lack
